### PR TITLE
10.6.0 Release

### PR DESCRIPTION
**Pull request by emilyrohrbough**
Co-authored-by: Blue F <blue@everblue.info>

### CODE REVIEW

File: content/guides/references/best-practices.md [hunk 1]
```diff
@@ -799,9 +799,9 @@ beforeEach(() => {
 
 One final question you should ask yourself is - is resetting the state even
 necessary? Remember, Cypress already automatically enforces
-[test isolation](/guides/core-concepts/test-isolation) by clearing state before
-each test. Make sure you are not trying to clean up state that is already
-cleaned up by Cypress automatically.
+[test isolation](//guides/core-concepts/writing-and-organizing-tests#Test-Isolation)
```
[the remark this answers is not in the document]
Comment (emilyrohrbough) [reply content/guides/references/best-practices.md:802]: 🤷🏻‍♀️ type though

File: content/_changelogs/10.6.0.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+## 10.6.0
+
+_Released 8/16/2022_
+
+**Features:**
+
+- Introduced a new Cypress experiment, experimentalSingleTabRunMode, for
```
Comment (emilyrohrbough) [inline content/_changelogs/10.6.0.md:7]:
```suggestion
- Introduced a new Cypress experiment, `experimentalSingleTabRunMode`, for
```

File: content/_changelogs/10.6.0.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  Component Testing. When enabled, all specs will execute in a single tab,
+  similar to the Component Testing alpha in Cypress 9. Addresses
```
Comment (emilyrohrbough) [inline content/_changelogs/10.6.0.md:9]:
```suggestion
  similar to the execution behavior observed in Component Testing alpha in Cypress 9, which can improve `run` mode performance, but can impact spec isolation and reliability on large test suites. Addresses
```

File: content/_changelogs/10.6.0.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  [#22353](https://github.com/cypress-io/cypress/issues/22353).
+- Added version as a variable that can be used in
+  CYPRESS_DOWNLOAD_PATH_TEMPLATE. Addresses
```
Comment (emilyrohrbough) [inline content/_changelogs/10.6.0.md:12]:
```suggestion
- Enhanced the [`CYPRESS_DOWNLOAD_PATH_TEMPLATE`](/guides/references/advanced-installation#Environment-variables) environment variable interpolation to accept and replace `${version}` to allow version-specific download paths to be honored. Addresses
```

File: content/_changelogs/10.6.0.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+- Fixed an issue where the Firefox browser was not downloading files to the
+  expected directory in Windows. Fixes
+  [#17896](https://github.com/cypress-io/cypress/issues/17896).
+- Fixed an issue where the unique session id logic was not persisting registered
+  session ids, which incorrectly allowed session ids to override previously used
+  session ids with different setups. Fixes
+  [#22381](https://github.com/cypress-io/cypress/issues/22381).
+- Fixed an issue where absolute path glob manipulation was incorrectly breaking
+  supportFile/spec glob patterns that contained deep instances of the working
```
Comment (emilyrohrbough) [inline content/_changelogs/10.6.0.md:27]:
```suggestion
  `supportFile` and `specPattern` glob patterns that contained deep instances of the working
```

**Review by emilyrohrbough — COMMENTED** [review]
👍🏻

**Review by mjhenkes — APPROVED** [review]

**Review by tbiethman — APPROVED** [review]

**Review by mschile — APPROVED** [review]

**Pull request merged**